models/lobby.model.js

```diff
@@ -1,11 +1,54 @@
 const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 
-const lobbySchema = new mongoose.Schema({
-    username: {
+const roundTypes = ['1', '2', '3'];
+
+const questionSchema = new Schema({
```

Comment:
questionSchema and commonQuestionSchema should be moved to a separate file and then imported to use in the lobbySchema.